graphistry/plotter.py

```diff
@@ -653,3 +655,27 @@ def _make_vgraph_dataset(self, edges, nodes, name):
         dataset = vgraph.create(elist, filtered_nlist, sources, dests, nodeid, node_map, name)
         dataset['encodings'] = encodings
         return dataset
+
+
+    def bolt(self, driver):
+        res = copy.copy(self)
+        res._bolt_driver = bolt_util.to_bolt_driver(driver)
+        return res
+
+
+    def cypher(self, query, params={}):
+        res = copy.copy(self)
+        driver = self._bolt_driver or PyGraphistry._config['bolt_driver']
+        with driver.session() as session:
+            bolt_statement = session.run(query, **params)
+            graph = bolt_statement.graph()
+            edges = bolt_util.bolt_graph_to_dataframe(graph)
+            nodes = bolt_util.bolt_graph_to_nodes(graph)
```

Comment:
Are we guaranteed to have nodes, e.g., what happens if someone does cypher ".... return r" instead of ".... return a, r, b"? (Not sure what `.graph()` does here..)

Reply:
It works implicitly. No nodes = no node info, but since we have edges, the graph is still visualized.